components/MonitoringStatusOverviewTable.vue

```diff
@@ -28,20 +28,35 @@
         </th>
         <td :class="$style.value">
           {{ item.itemValue.value }}
-          <span :class="$style.unit">{{ $t(item.itemValue.unit) }}</span>
+          <span v-if="item.itemValue.unit != '%'" :class="$style.unit">{{
```

Comment:
暗黙型変換を伴わない比較演算子を使った方がよい
```suggestion
          <span v-if="item.itemValue.unit !== '%'" :class="$style.unit">{{
```
また将来的には `utils/formatMonitoringStatus.ts` 内での単位定義時に `translatable` （翻訳可能）かどうかのプロパティを設け、その値でv-ifするようにしたほうが良いと思う。